src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/Service/WebhookReachabilityChecker.php

```diff
@@ -88,4 +98,9 @@ public function check(string $url, string $secret): UrlReachabilityStatus
             }
         }
     }
+
+    private function isRedirectResponse(int $statusCode): bool
+    {
+        return in_array($statusCode, [301, 302, 303, 307, 308]);
```

Comment:
```suggestion
        return statusCode >= 300 && statusCode < 400;
```
Future proof and simpler ;)